_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 {% assign identifier = site.data.usa_identifier.identifier_data[0] %}
 
-<div class="usa-identifier">
+<footer class="usa-identifier">
```

Comment:
I know on the other ones that I've worked on the footer has a class of `usa-footer` instead, and it contains the `usa-identifier`, and so I'm wondering it makes sense to do the same here? That way the footer can hold the identifier plus anything else it might need.

Reply:
that makes sense!